cache/network/upload.go

```diff
@@ -69,3 +71,27 @@ func validateKey(key string, logger log.Logger) (string, error) {
 	}
 	return key, nil
 }
+
+func logResponseMessage(response acknowledgeResponse, logger log.Logger) {
+	if response.Message == "" || response.Severity == "" {
+		return
+	}
+
+	var loggerFn func(format string, v ...interface{})
+	switch response.Severity {
+	case "debug":
+		loggerFn = logger.Debugf
+	case "info":
+		loggerFn = logger.Infof
+	case "warning":
+		loggerFn = logger.Warnf
+	case "error":
+		loggerFn = logger.Errorf
+	default:
+		loggerFn = logger.Printf
+	}
+
+	loggerFn("\n")
+	loggerFn(response.Message)
+	loggerFn("\n")
+}
```

Comment:
This is interesting as the server dictates what kind of message it returns. I have not done this in the past as I was just printing out information received from the server.

Do we expect that the server returns all of these message severity? I mean at the moment I can only see three:
- Server returns nothing -> nothing is printed -> everything is fine
- Server returns something -> it is printed as an info in the log 
- Server returns an error -> the world is collapsing